components/resource-creation/ResourcePanel.tsx

```diff
@@ -43,6 +82,17 @@ export default function ResourcePanel() {
           }}
         >
           <ResourceSelection />
+          <Center>or</Center>
+          <Center>
+            <Button
+              aria-label="Add New Custom Resource"
+              onClick={() => setIsNewResourceModalOpen(true)}
+              variant="outline"
+            >
```

Comment:
Would you oppose making the "Add New Custom Resource" option as an option in the dropdown menu rather than a button? Might make it look more similar to the original design but I also like the button!

Reply:
I prefer the button to a dropdown menu option so that we can more easily differentiate the two options for resource creation (from requirements versus from manually adding a resource with a modal). I think it would be more difficult to differentiate the two options if we had it in the dropdown because then we would need to also somehow make it clear that the other resources came from the data requirements